controllers/hcloudmachine_controller.go

```diff
@@ -157,6 +159,36 @@ func (r *HCloudMachineReconciler) Reconcile(ctx context.Context, req reconcile.R
 			res = reconcile.Result{}
 			reterr = errors.Join(reterr, err)
 		}
+
+		readyReason := conditions.GetReason(machineScope.HCloudMachine, clusterv1.ReadyCondition)
+		readyMessage := conditions.GetMessage(machineScope.HCloudMachine, clusterv1.ReadyCondition)
+
+		duration := time.Since(startReconcile)
```

Comment:
do we have a good reason to keep this in the code block that you added in hcloudmachine_controller? If not, I'd prefer to remove it. We can always add it for testing something specific